tests/tck/features/optimizer/PrunePropertiesRule.feature

```diff
@@ -21,12 +21,12 @@ Feature: Prune Properties rule
       | 33            |
       | 41            |
     And the execution plan should be:
-      | id | name           | dependencies | operator info                                                                                              |
-      | 8  | Project        | 4            |                                                                                                            |
-      | 4  | AppendVertices | 3            | {  "props": "[{\"props\":[\"age\"],\"tagId\":9}]" }                                                        |
-      | 3  | Traverse       | 7            | {  "vertexProps": "", "edgeProps": "[{\"props\":[\"_dst\", \"_rank\", \"_type\", \"_src\"],\"type\":3}]" } |
-      | 7  | IndexScan      | 2            |                                                                                                            |
-      | 2  | Start          |              |                                                                                                            |
+      | id | name           | dependencies | operator info                                                                                    |
+      | 8  | Project        | 4            |                                                                                                  |
+      | 4  | AppendVertices | 3            | {  "props": "[{\"props\":[\"age\"],\"tagId\":9}]" }                                              |
+      | 3  | Traverse       | 7            | {  "vertexProps": "", "edgeProps": "[{\"props\":[\"_dst\", \"_rank\", \"_type\"],\"type\":3}]" } |
```

Comment:
I think we need `_dst` of edge only in this case.

Reply:
In order to distinguish the same edge in the path, _type and _rank also need to be

Reply:
ok